core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/MetricsServiceImpl.java

```diff
@@ -142,12 +142,18 @@ public int hashCode() {
     private DataAccess dataAccess;
 
     public void startUp(Session session, String keyspace, boolean resetDb) {
+        startUp(session, keyspace, resetDb, true);
+    }
+
+    public void startUp(Session session, String keyspace, boolean resetDb, boolean createSchema) {
         SchemaManager schemaManager = new SchemaManager(session);
         if (resetDb) {
             schemaManager.dropKeyspace(keyspace);
         }
-        // This creates/updates the keyspace + tables if needed
-        schemaManager.createSchema(keyspace);
+        if (createSchema) {
+            // This creates/updates the keyspace + tables if needed
```

Comment:
It may be that this change  is not required given other startup changes.  I don't really see how it hurts anything, though.